### ISSUE TITLE
Links broken on cards

### ISSUE THREAD

**Issue by Unisergius** [onelikeandidie/meetballs.org#42]
source links seem to be broken 

Examples: 

https://www.facebook.com/events/451186503902037/)%20-%20Flutter%20Installation%20Part

https://fb.me/e/3jkI9ydYI)%20-%20Introduction%20to%20a%20VueJ

https://github.com/Unisergius/flutter-fccikea-3-2023)%20-%20Session%20from%20Flutter%20Faro

**Comment by onelikeandidie** [comment]
I noticed that the other day, I think the problem is when I imported the projects from the original README.md. I think the main culprit is this function on the import.

https://github.com/onelikeandidie/meetballs.org/blob/70c56dea11707899cdac47cb0b0f09b3e12937a6/app/Libraries/Importer/Parser.php#L159

Once this is fixed to work out the links correctly we can rerun the imports and it will fix the broken links. Either that or I go through and fix each link manually :skull:
